examples/accel/firmware.c

```diff
@@ -77,7 +77,14 @@ enum BuiltinKernelId : uint16_t
   POCL_CDBI_MUL_I16 = 15,
   POCL_CDBI_STREAMOUT_I32 = 16,
   POCL_CDBI_STREAMIN_I32 = 17,
-  POCL_CDBI_LAST = 18,
```

Comment:
Add a mention to the comment of this enum that all of these are experimental DBKs and are subject to change in future PoCL releases without deprecation periods.

Reply:
done